compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -513,6 +513,23 @@ class ComposeScene internal constructor(
         snapshotChanges.perform()
     }
 
+    internal fun hitTestInteropView(position: Offset): Boolean {
+        // TODO: temporary solution copying control flow from [processPress]
```

Comment:
We should add an explanation why it is temporary

Reply:
Done